### DISCUSSION TITLE
[Kavita] Allow page navigation with mouse wheel

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2539] [Ideas]
### Idea Description

Posted by maws
· Feb 2022
· 41 Upvotes
https://feats.kavitareader.com/posts/78/allow-page-navigation-with-mouse-wheel

Description

In the reader, allow scrolling past the end of a page to switch to the next (or previous) page. This is implemented in several other readers, like for example Calibre.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by zuilintan** [comment]
Good Idea

**Comment by majora2007** [comment]
Please clarify which reader, we have 3. I assume the image reader.

**Comment by zuilintan** [comment]
> Please clarify which reader, we have 3. I assume the image reader.

Yes，
In addition, about comic readers, I found a great web page script: 
[https://sleazyfork.org/zh-CN/scripts/374903-comicread](url)

Unfortunately, it is not compatible with us,
Can you solve it?

**Reply by scare376** [reply]
with how kavita works i dont think there would be a way to make that script work without changing how the reader functions. instead you should create a new discussion about including the functionality of the script within the reader itself.
